### PR TITLE
feat: Responsiveness for small screen sizes

### PR DESCRIPTION
**Pull request by tushartiwari7**
This includes the media queries and other code to make the website responsive for small screen sizes.

Things added in this PR
 - Added Responsiveness using media queries and layouts. 
 - Getting Started Page
 - Color Palette
 - Installation Guides

### CODE REVIEW

**Comment by tushartiwari7** [comment]
Things added in this PR
 - Responsiveness
 - Getting Started Page
 - Color Palette
 - Installation Guides

File: index.html [hunk 1]
```diff
@@ -10,6 +10,9 @@
 </head>
 <body class="grid fullheight">
   <header class="flex flex-center fullwidth p-mid fs-l" id="header">
```
Comment (AnkitKarnAK) [inline index.html]: use the same naming convention, prefer p-md or padding-md

Comment (AnkitKarnAK) [inline index.html]: it should be full-width

File: index.html [hunk 2]
```diff
@@ -23,10 +26,19 @@
     <div class="brand">
       <img src="https://picsum.photos/300/150" alt="Elevate UI Logo" class="brand-logo">
       <h3 class="fs-xl p-smaller brand-name heading">Elevate UI</h3>
```
Comment (AnkitKarnAK) [inline index.html]: either use short forms or full forms but not both for sizes

File: index.html [hunk 3]
```diff
@@ -23,10 +26,19 @@
...
+      <button class="btn btn-float closeIcon flex flex-center rounded-circle btn-outline-primary p-small fs-m pos-abs"
```
Comment (AnkitKarnAK) [inline index.html]: don't use camel case for css classes

File: index.html (hunk 2 above)
Comment (AnkitKarnAK) [inline index.html]: may be don't use brand-name class just for aligning to center

File: index.html [hunk 4]
```diff
@@ -23,10 +26,19 @@
...
+      onclick="closeSidebar()"
+      >
+        <i class="bi bi-x"></i>
+      </button>
     </div>    
     <ul class="m-mid fs-m list-reset flex flex-col" >
-      <li class="mx-mid variant my-smaller transition component-name pl-smaller fw-light" >Get Started</li>
-      <li class="mx-mid variant my-smaller transition component-name pl-smaller fw-light" >Installation</li>
+      <li class="mx-mid variant my-smaller transition component-name pl-smaller fw-light"
```
Comment (AnkitKarnAK) [inline index.html]: don't use both xs or smaller

File: layout/App.css [hunk 5]
```diff
@@ -26,3 +27,87 @@ nav {
 .fullheight {
```
Comment (AnkitKarnAK) [inline layout/App.css]: use - to seperate

File: layout/App.css [hunk 6]
```diff
@@ -26,3 +27,87 @@ nav {
...
+    height: fit-content;
+  }
+  
+  nav > h4 {
+    display: none;
+  }
+  nav > ul.list {
+    padding: 0;
+    display: flex;
+    flex-direction: row;
```
Comment (AnkitKarnAK) [inline layout/App.css:75]: row is default

File: layout/App.css [hunk 7]
```diff
@@ -26,3 +27,87 @@ nav {
...
+    display: block;
+  }
+
+  button.closeIcon {
+    display: flex;
+  }
+
+  aside.sidebar {
+    position: absolute;
+    z-index: 10;
```
Comment (AnkitKarnAK) [inline layout/App.css]: don't use z-index:10 if this is only layer

File: layout/App.css [hunk 8]
```diff
@@ -26,3 +27,87 @@ nav {
...
+    background: white;
```
Comment (AnkitKarnAK) [inline layout/App.css:106]: i think not needed unless you're changing background for different screen sizes

File: pages/getStarted/getStarted.css [hunk 9]
```diff
@@ -0,0 +1,38 @@
...
+}
+
+.feat-card-wrapper {
+  justify-content: space-evenly;
+}
+
+.color {
+  border: 1px solid var(--text-color100);
+  text-align: center;
+}
```
Comment (AnkitKarnAK) [inline pages/getStarted/getStarted.css]: class name doesn't justifies css properties

File: pages/getStarted/getStarted.css [hunk 10]
```diff
@@ -0,0 +1,38 @@
...
+.color:nth-child(2) {
+  background-color: var(--bg-primary);
+}
+.color:nth-child(3) {
+  background-color: var(--bg-secondary);
+}
+.color:nth-child(4) {
+  background-color: var(--text-color);
+  color: var(--bg-secondary);
+}
```
Comment (AnkitKarnAK) [inline pages/getStarted/getStarted.css]: same as above. Also, add utility classes instead of using nth child for each

**Review by Raghav888 — APPROVED** [review]
Approved

File: layout/App.css (hunk 8 above)
Comment (tushartiwari7) [reply layout/App.css:106]: My sidebar was becoming transparent without declaring any background-color so I put white here.
sidebar_error

File: layout/App.css (hunk 6 above)
Comment (tushartiwari7) [reply layout/App.css:75]: i have set it to column in default(desktop) view so when media-queries gets activated I want it to be row.

File: layout/App.css (hunk 7 above)
Comment (tushartiwari7) [reply layout/App.css]: done

File: index.html (hunk 2 above)
Comment (tushartiwari7) [reply index.html]: done

Comment (tushartiwari7) [reply index.html]: done

File: pages/getStarted/getStarted.css (hunk 10 above)
Comment (tushartiwari7) [reply pages/getStarted/getStarted.css]: done

File: pages/getStarted/getStarted.css (hunk 9 above)
Comment (tushartiwari7) [reply pages/getStarted/getStarted.css]: done! using `color-palette-item` now

**Review by ashishnnnnn — APPROVED** [review]
Code seems pretty much good to merge :+1:

**Pull request merged**